test-data/unit/check-typevar-values.test

```diff
@@ -496,12 +496,18 @@ def outer(x: T) -> T:
 [out]
 
 [case testClassMemberTypeVarInFunctionBody]
-from typing import TypeVar
+from typing import TypeVar, List
 class C:
     T = TypeVar('T', bound=int)
     def f(self, x: T) -> T:
-        A = C.T
-        return x
+        L = List[C.T] # a valid type alias
```

Comment:
Oh!  How about 
* "Type variable 'C.T' is not a valid expression"
* "Type variable 'C.T' cannot be used as an expression"
?